### DISCUSSION TITLE
I love achievements

### DISCUSSION THREAD

**Discussion by machacjozef** [community/community#21096] [Other Feature Feedback, Questions, & Ideas]
please more achievements

**Comment by radar-blip** [comment]
You cannot mark your own answers, stop.

**Reply by pffy** [reply]
^^

**Comment by thuongtruong1009** [comment]
Hi @machacjozef 
Below are some of my achievements 
![image](https://user-images.githubusercontent.com/71834167/180588884-8e4623cc-b954-46a3-bb9d-db881f332d2b.png)
 
See at https://github.com/thuongtruong1009

**Comment by RyanMan56** [comment]
I'm loving the YOLO achievement - my bad practices feel very called out!
